jpo-ode-svcs/src/main/java/us/dot/its/jpo/ode/traveler/TimController.java

```diff
@@ -283,6 +276,15 @@ public TimController(OdeProperties odeProperties) {
       OdeTimData odeTimData = new OdeTimData(timMetadata, timDataPayload);
       timProducer.send(odeProperties.getKafkaTopicOdeTimBroadcastPojo(), null, odeTimData);
 
+      // Short circuit
+      // If the TIM has no RSU/SNMP or SDW structures, we are done
+      if (travelerInputData.getRsus() == null && travelerInputData.getSnmp() == null
+            && travelerInputData.getSdw() == null) {
```

Comment:
I think the if statement should be like this:
`if ((travelerInputData.getRsus() == null || travelerInputData.getSnmp() == null)
            && travelerInputData.getSdw() == null)`